save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/chart/PieChartBuilder.kt

```diff
@@ -14,15 +14,15 @@ import kotlin.random.Random
  * @return ReactElement
  */
 @Suppress("MAGIC_NUMBER")
-fun RBuilder.pieChart(
+fun ChildrenBuilder.pieChart(
     data: Array<DataPieChart>,
-    handler: RHandler<PieChartProps> = {},
-) = child(PieChart::class) {
-    attrs.data = data
-    attrs.animate = false
-    attrs.segmentsShift = 0
-    attrs.viewBoxSize = intArrayOf(100, 100)
-    attrs.radius = 50
+    handler: ChildrenBuilder.(props: PieChartProps) -> Unit = {},
+) = PieChart::class.react.create {
```

Comment:
I don't think it should be `create`, but rather extension `ElemenType.invoke` from `ChildrenBuilder` should be used